src/Entity/Hashtag.php

```diff
@@ -5,28 +5,37 @@
 class Hashtag extends AbstractEntity
 {
     /**
-     * {@inhertDoc}.
+     * @return string
+     */
+    private function getEntityKey()
+    {
+        return '#';
```

Comment:
Is it worth saving this as a `const` on the class? `const DEFAULT_ENTITY_KEY = '#';`